src/main/resources/hazelcast-config-schema.json

```diff
@@ -1373,20 +1373,535 @@
+                },
+                "trusted-interfaces": {
+                  "type": "array",
+                  "description": "Includes IP addresses of trusted members. When a node wants to join to the cluster, its join request will be rejected if it is not a trusted member. You can give an IP addresses range using the wildcard (*) on the last digit of IP address (e.g. 192.168.1.* or 192.168.1.100-110).",
+                  "items": {
+                    "type": "string"
+                  }
+                }
+              }
+            },
+            "tcp-ip": {
+              "type": "object",
+              "additionalProperties": false,
+              "properties": {
+                "enabled": {
+                  "type": "boolean",
+                  "default": false,
+                  "description": "Specifies whether the TCP/IP discovery is enabled or not."
+                },
+                "connection-timeout-seconds": {
+                  "type": "integer",
+                  "minimum": 0,
+                  "default": 5,
+                  "description": "The maximum amount of time Hazelcast is going to try to connect to a well known member before giving up. Setting it to a too low value could mean that a member is not able to connect to a cluster. Setting it to a too high value means that member startup could slow down because of longer timeouts (e.g. when a well known member is not up). Increasing this value is recommended if you have many IPs listed and the members cannot properly build up the cluster."
+                },
+                "required-member": {
+                  "type": "string",
+                  "description": "IP address of the required member. Cluster will only be formed if the member with this IP address is found."
+                },
+                "member": {
+                  "type": "string",
+                  "description": "IP address(es) of one or more well known members. Once members are connected to these well known ones, all member addresses will be communicated with each other. You can also give comma separated IP addresses using the \"members\" element or list the members with the \"member\" element under \"member-list\"."
+                },
+                "members": {
+                  "type": "string",
+                  "description": "Comma separated IP addresses of one or more well known members."
+                },
+                "member-list": {
+                  "type": "array",
+                  "description": "Sequence of IP address(es) of one or more well known members.",
+                  "items": {
+                    "type": "string"
+                  }
+                },
+                "interface": {
+                  "type": "string",
+                  "description": "IP address(es) of one or more well known members."
+                }
+              }
+            },
+            "aws": {
```

Comment:
It would be good to detail out the other properties too, based on the documentation. Same for gcp, azure, kubernetes, eureka and discovery-strategies.

Reply:
I chose to leave them out on purpose as they depend on the plugin version that's used. I believe this is the reason that original example left these out.